### DISCUSSION TITLE
What does AE in GitHub AE stand for?

### DISCUSSION THREAD

**Discussion by niranjanviladkar** [community/community#59931] [Enterprise] [answered]
### Select Topic Area

General

### Body

What is the full form of AE in GitHub AE?
What does AE in GitHub AE stand for?

I could not find any [previous answers](https://github.com/orgs/community/discussions?discussions_q=What+does+AE+in+GitHub+AE+stand+for%3F) to this question.

**Comment by nirav-gajera** [comment]
"AE" stands for "GitHub Enterprise Server." GitHub Enterprise Server (GitHub AE) is a self-hosted version of GitHub that organizations can deploy on their own infrastructure. It provides the same functionality as the cloud-based GitHub platform but allows organizations to have more control over their repositories and data by hosting it on their own servers.

**Reply by niranjanviladkar** [reply]
Thanks @nirav-gajera for your reply! Shouldn't then it be acronym-ed `GitHub ES`?
I mean, does AE come from a different language?

**Reply by nirav-gajera** [reply, marked as answer]
It is not related to any specific language but rather refers to a specific version or edition of GitHub called "GitHub Enterprise."

On the other hand, "ES" stands for "Enterprise Server," indicating that it is a server version of GitHub tailored for enterprises. So, GitHub AE and GitHub Enterprise Server essentially refer to the same self-hosted enterprise version of GitHub with different acronyms.

**Reply by niranjanviladkar** [reply]
Thanks @nirav-gajera!

**Comment by ShaunSoCo** [comment]
Actually GitHub AE is in closed beta.  It is a GitHub-hosted private cloud version of GitHub Enterprise.  All public documents referring to GHAE have been removed except for the Microsoft stuff:
https://learn.microsoft.com/en-us/entra/identity/saas-apps/github-ae-provisioning-tutorial

**Reply by naikrovek** [reply]
This is the info I was hoping to find, thank you.

**Comment by jklukas** [comment]
Another possibility is that "AE" originally meant "Azure Edition", but I can't find anything that confirms that.
